src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -151,8 +161,8 @@ <h4>{{ 'artemisApp.exerciseAssessmentDashboard.gradingInstructions' | artemisTra
         ></jhi-structured-grading-instructions-assessment-layout>
     </div>
 
-    <div class="col-12 text-center" *ngIf="tutorParticipationStatus === NOT_PARTICIPATED && !isTestRun">
-        <button class="btn btn-primary guided-tour-instructions-button" (click)="readInstruction()">
+    <div class="col-12 text-center" style="margin-top: 20px; margin-bottom: 40px" *ngIf="tutorParticipationStatus === NOT_PARTICIPATED && !isTestRun">
```

Comment:
I think it is worth considering to express this css with bootstrap classes, see https://getbootstrap.com/docs/5.0/utilities/spacing/ (not all of it works from my experience, but most does), so we don't need to mix boostrap with inline css

Reply:
good suggestion, it's done now. 👍 @MDK5